pkg/types/pool.go

```diff
@@ -25,7 +26,8 @@ var (
 )
 // Pool defines cpupool
 type Pool struct {
-	CPUs string `yaml:"cpus"`
+	CPUStr string        `yaml:"cpus"`
```

Comment:
you don't need to store the  CPUs in the struct just to pars them
use a temporary inline struct for parsing